Packs/CommonScripts/Scripts/diplayMappedFields/diplayMappedFields.py

```diff
@@ -14,50 +21,30 @@ def extract_keys_with_values(obj, parent_key=''):
     return items
 
 
-def format_data_to_table(items):
-    """Formats the extracted data into a table format"""
-    table = '| Field Name | Value |\n'
-    for key, value in items:
-        table += f'| {key} | {value} |\n'
-    return table
-
-
-def convert_to_html(markdown):
-    html = "<table style='border-collapse: collapse;'>"
-    lines = markdown.split("\n")
-    for line in lines[1:]:
-        html += "<tr>"
-        columns = line.split("|")
-        for i, column in enumerate(columns):
-            if column.strip():
-                if i != 1:
-                    html += f"<td style='font-family: Lato, Assistant, sans-serif; font-weight: 600; \
-                    font-size: 12px; text-align: left;  \
-                    padding: 1px 0px 0px; margin: 0px 5px 0px 0px; contrast: 4.95; ֿֿֿ\
-                    color: var(--xdr-on-background) ;'>{column.strip()}</td>"
-                else:
-                    html += f"<td style='font-family: Lato, Assistant, sans-serif; font-weight: 600; \
-                    font-size: 12px; text-align: left; \
-                    padding: 1px 0px 0px; margin: 0px 5px 0px 0px; contrast: 4.95; \
-                    color: var(--xdr-on-background-secondary);'>{column.strip()}</td>"
-        html += "</tr>"
-    html += "</table>"
-    return html
-
-
-def remove_empty_rows(table):
-    # Split the table into rows
-    rows = table.split("\n")
+def format_data_to_rows(items):
+    """
+    Formats the extracted data into rows.
+    """
+    return [f'{key}|{value}' for key, value in items]
 
-    # Filter out the rows that contain empty dictionaries
-    filtered_rows = [row for row in rows if not (
-        "{}" in row or "[{}]" in row or "[{}, {}]" in row or "[{}, {}, {}]" in row or "0001-01-01T00:00:00Z" in row or " \
```

Comment:
``` 
" \
    containmentsla"
```
 is a bug, since it's actually a string with a new line and spaces and not just `"containmentsla"`